src/extendables/User.ts

```diff
@@ -29,6 +29,27 @@ export default class extends Extendable {
 			];
 		}
 
+		if (monster.consumedItem) {
```

Comment:
i dont think you can do this here unless you pass a quantity to the function, and it doesnt stop you from having to basically check this again in the kill function since you need to do removal of items there anyways